backend/app.js

```diff
@@ -0,0 +1,8 @@
+var express = require('express');
+var app = express();
```

Comment:
Don't use `var`.
Use `const` or `let`.
```suggestion
const app = express();
```

https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var